lib/presentation/home_screen/home_screen.dart

```diff
@@ -21,27 +21,68 @@ class _HomeScreenState extends State<HomeScreen> {
     mapController = controller;
   }
 
-
   @override
   Widget build(BuildContext context) {
-
     return Scaffold(
       appBar: AppBar(
         title: Text(widget.title),
       ),
-      drawer: MyDrawer(
+      drawer: const MyDrawer(
         userName: 'Help A Paw',
         userImage: 'assets/images/paww.png',
       ),
-      body: GoogleMap(
-        onMapCreated: _onMapCreated,
-        initialCameraPosition: CameraPosition(
-          target: _center,
-          zoom: 11.0,
-        ),
-        zoomControlsEnabled: false,
-        myLocationEnabled: true,
+      body: Stack(
+        children: [
+          GoogleMap(
+            onMapCreated: _onMapCreated,
+            initialCameraPosition: CameraPosition(
+              target: _center,
+              zoom: 11.0,
+            ),
+            myLocationButtonEnabled: false,
+            myLocationEnabled: true,
+            zoomControlsEnabled: false,
+          ),
+          Positioned(
```

Comment:
Hm, doesn't Google Maps have this button built-in?

Reply:
i forgot Googlemaps have built in button, already change it in the new Commit